plugins/.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+# This config is deprecated.
+# Please use the root .pre-commit-config.yaml file which now includes all necessary hooks.
+# This file is kept only for backward compatibility and may be removed in the future.
```

Comment:
I'd prefer just removing this file if not used

Reply:
Fixed